specification/recoveryservicesbackup/resource-manager/readme.go.md

```diff
@@ -13,12 +13,22 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2019-06
```

Comment:
Any reason that Java version is not updated as well. Looks like Java is lagging behind for quite a few package versions.

Reply:
This is added on demand. This api-version is requested by an email request, when I am releasing the go SDK, I noticed that the go config file does not have this api-version, therefore i added this one in this PR.
I suppose the java SDK owner will spot the same problem and submit his own PR for java new api-version when he is releasing the java SDK for the same request

Reply:
Ok, can you send an email and notify respective SDK owners? Thanks.